lab_2/main.py

```diff
@@ -43,7 +78,20 @@ def get_text_vector(original_text: list, language_profiles: dict) -> list or Non
     :param original_text: any tokenized text
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
-    pass
+    if not isinstance(original_text, list) or not isinstance(language_profiles, dict) or not language_profiles:
+        return None
+
+    features = get_language_features(language_profiles)
+    vector_list = []
+    for word in features:
+        if word in original_text:
+            for lang_profile in language_profiles.values():
+                if word in lang_profile.keys():
+                    vector_list.append(lang_profile[word])
```

Comment:
there is a way to get a key if it exists or 0 if it doesn't

Reply:
google what arguments `get` needs
this is your task till Friday

Reply:
dict.get method gets a value from the dictionary by its key

Reply:
Changed to:
    for word in features:
        for lang_profile in language_profiles.values():
            if word in lang_profile:
                vector_list.append(lang_profile.get(word) if word in original_text else 0)